nixos/modules/services/desktops/pipewire/easyeffects.nix

```diff
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.pipewire.easyeffects;
+in
+{
+  meta.maintainers = with lib.maintainers; [ starcraft66 ];
+
+  options = {
+    services.pipewire.easyeffects = {
+      enable = lib.mkEnableOption "Audio effects for PipeWire applications.";
+
+      package = lib.mkOption {
+        type = lib.types.package;
+        default = pkgs.easyeffects;
+        defaultText = lib.literalExpression "pkgs.easyeffects";
+        description = ''
+          The easyeffects derivation to use.
+        '';
+      };
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    assertions = [
+      {
+        assertion = config.services.pipewire.enable && !config.hardware.pulseaudio.enable;
+        message = "Easyeffects requires Pipewire to be enabled and is not compatible with PulseAudio.";
+      }
+    ];
+
+    environment.systemPackages = [ cfg.package ];
+    systemd.packages = [ cfg.package ];
+
+    systemd.user.services.easyeffects = {
```

Comment:
I am not convinced this belongs to Nixpkgs. A module named after a program IMO implies some degree of officiality and even being the recommended way to install the software, but this service has nothing to do with upstream Pipewire.

Reply:
I'm not aware of a better way to install easyeffects on NixOS and have it run in the background by default.

> this service has nothing to do with upstream Pipewire

Are you suggesting that this module should instead reside somewhere else in nixpkgs than `services/desktops/pipewire`? I'd be happy to work with easyeffects developers to find out which way they prefer to have the software installed on NixOS but they have no official NixOS-related documentation so I'm just trying to integrate something clean there. I'd be happy to change it if they aren't happy but there is no word from them.

Reply:
It would probably fit better to `programs` section. But the primary issue for me is the coupling of “enable” with “install a service that will autostart”.